### PR TITLE
BD-2213 Audience Sync to Facebook

### PR DESCRIPTION
**Pull request by rachel-feinberg**
# Pull Request/Issue Resolution

Closes #BD-2213

#### Is this change associated with a Braze feature/product release?
- [ ] Yes (**Insert Feature Release Date Here**)
- [ ] No

---

<details>
<summary>✔️ Pull Request Checklist</summary>
<br>

- [ ] Check that you haven't removed any images (replacing an image with an updated one of the same name is fine), as this breaks the French site
- [ ] Check that all links work.
- [ ] Ensure you have completed [our Contributors License Agreement](https://www.braze.com/docs/cla/).
- [ ] Tag @josh-mccrowell-braze and @KellieHawks as a reviewer when your work is **done and ready to be reviewed for merge**. Are you an internal product manager? Reference the internal reviewing chart to tag the appropriate reviewer.
- [ ] Tag others as reviewers as necessary.
- [ ] If you have modified any links, be sure to add redirects to `assets` > `js` > `broken_redirect_list.js`

</details>

<details>
<summary>⭐ Helpful Wiki Shortcuts</summary>
<br>

- [Writing Style Guide](https://docs.google.com/document/d/e/2PACX-1vTluyDFO3ZEV7V6VvhXE4As_hSFwmnFFdU9g6_TrAYTgH1QmbRoEDDdn5GzKAB9vdBbIdyiFdoaJcNk/pub)
- [Image Style Guide](https://docs.google.com/document/d/e/2PACX-1vRJSkwcjmjrTfLDagZccLpOMMyh5NN5SXRZSjz12cRAHbX4OrUmhvCmYpf_p5YB-9r4_jSOQLkicQIH/pub)
- [Styling Test Page](https://www.braze.com/docs/home/styling_test_page/)

</details>

<details>
<summary>❗ ATTN: For PR Reviewers</summary>
<br>

- [ ] Read our [Reviewing a PR page](https://github.com/Appboy/braze-docs/wiki/Reviewing-a-PR) for more on our reviewing suggestions.
- [ ] Read our [Previewing Documentation page](https://github.com/braze-inc/braze-docs/wiki/Previewing-and-Testing-Documentation) to see how to check the deployment.
  - [ ] Preview all changes in the linked Vercel environment by clicking the preview link in the vercel-bot comment in your PR.
</details>

### CODE REVIEW

File: _docs/_partners/canvas_steps/facebook_audience_sync.md [hunk 1]
```diff
@@ -124,6 +120,10 @@ The **History** tab of the custom audience in the Facebook Audience Manager will
 
 ![Audience details and the History tab for a given Facebook audience that includes an Audience History table with columns for the activity, activity details, items changed, and the date and time.][9]{: style="max-width:80%;"}
 
+## Migrating to Meta Work Accounts
```
Comment (josh-mccrowell-braze) [inline _docs/_partners/canvas_steps/facebook_audience_sync.md:123]:
```suggestion
## Migrating to Meta work accounts
```

Headers should be in sentence case.

File: _docs/_partners/canvas_steps/facebook_audience_sync.md [hunk 2]
```diff
@@ -124,6 +120,10 @@ The **History** tab of the custom audience in the Facebook Audience Manager will
...
+
+Starting in July 2023, Meta is rolling out Meta work accounts to a small set of businesses who are interested in adopting this new account type. If you have a Business Account integrated with Braze, please ensure you disconnect and reconnect to the [Facebook partner page]({{site.baseurl}}/partners/canvas_steps/facebook_audience_sync#step-1-connect-to-facebook) with your Business Account in order to perserve this implementation and not disrupt any active Canvases.
```
Comment (josh-mccrowell-braze) [inline _docs/_partners/canvas_steps/facebook_audience_sync.md:125]:
```suggestion
Starting in July 2023, Meta is rolling out Meta work accounts to a small set of businesses who are interested in adopting this new account type. If you have a business account integrated with Braze, please ensure you disconnect and reconnect to the [Facebook partner page]({{site.baseurl}}/partners/canvas_steps/facebook_audience_sync#step-1-connect-to-facebook) with your business account in order to preserve this implementation and not disrupt any active Canvases.
```

Comment (josh-mccrowell-braze) [reply _docs/_partners/canvas_steps/facebook_audience_sync.md:125]: If you haven't done so yet, it'd a good idea to make sure you have a spellcheck extension loaded up in your VS Code! Mine saves my life.

Comment (KellieHawks) [inline _docs/_partners/canvas_steps/facebook_audience_sync.md:125]:
```suggestion
Starting in July 2023, Meta is rolling out Meta work accounts to a small set of businesses who are interested in adopting this new account type. If you have a Business Account integrated with Braze, ensure you disconnect and reconnect to the [Facebook partner page]({{site.baseurl}}/partners/canvas_steps/facebook_audience_sync#step-1-connect-to-facebook) with your Business Account in order to perserve this implementation and not disrupt any active Canvases.
```

**Review by josh-mccrowell-braze — APPROVED** [review]
Looks great! Just highlighted a few small things.

File: _docs/_partners/canvas_steps/facebook_audience_sync.md (hunk 2 above)
Comment (KellieHawks) [reply _docs/_partners/canvas_steps/facebook_audience_sync.md:125]: We try to not use "please" in our docs, except for customer copy examples. TY!

**Review by KellieHawks — CHANGES_REQUESTED** [review]
Looks great! One minor comment.

File: _docs/_partners/canvas_steps/facebook_audience_sync.md (hunk 2 above)
Comment (rachel-feinberg) [reply _docs/_partners/canvas_steps/facebook_audience_sync.md:125]: I'll do that! I just realized there's no spellcheck.

**Review by KellieHawks — APPROVED** [review]

**Comment by KellieHawks** [comment]
Great work!

**Pull request merged**